tools/sqlite3_api_wrapper/sqlite3_api_wrapper.cpp

```diff
@@ -501,6 +504,31 @@ const unsigned char *sqlite3_column_text(sqlite3_stmt *pStmt, int iCol) {
 			// not initialized yet, convert the value and initialize it
 			entry.data = unique_ptr<char[]>(new char[val.str_value.size() + 1]);
 			memcpy(entry.data.get(), val.str_value.c_str(), val.str_value.size() + 1);
+			entry.data_len = val.str_value.length();
+		}
+		return (const unsigned char *)entry.data.get();
+	} catch (...) {
+		// memory error!
+		return nullptr;
+	}
+}
+
+const void *sqlite3_column_blob(sqlite3_stmt *pStmt, int iCol) {
+	Value val;
+	if (!sqlite3_column_has_value(pStmt, iCol, LogicalType::BLOB, val)) {
```

Comment:
Could you add tests for `sqlite3_column_blob`, `sqlite3_bind_blob` and `sqlite3_bind_zeroblob` to `test_sqlite3_api_wrapper.cpp`?

Reply:
Sure